bitrise.yml

```diff
@@ -47,15 +47,15 @@ workflows:
     - restore-spm-cache@1:
         is_always_run: true
     - git::https://github.com/DamienBitrise/bitrise-test-plan-sharder.git@master:
-          title: Bitrise Test Plan Sharder
```

Comment:
The Python `ruamel.yaml` reformats `bitrise.yml`. Let me format the file so that the diffs in the future is easier.